src/_product-updates/2024-08-10-feedbacks-gathering.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Documentation feedback 
+tags: [New]
+image: /images/changelog/feedback-button.png
+---
+
+![feedback-button.png](/images/changelog/feedback-button.png)
+
+Because feedback from real-life users is the best way to improve your documentation, our last update introduces a new "Give feedback" button.
+
+The `x-feedbackLink` object can be added directly in your API definition file.  
```

Comment:
```suggestion
The `x-feedbackLink` object can be added directly in the `info` object of your API definition file. 
```

(I agree with less implementation details, but I think mentioning the info object here is still important)